stripe-core/src/main/java/com/stripe/android/core/utils/FeatureFlags.kt

```diff
@@ -5,17 +5,20 @@ import com.stripe.android.core.BuildConfig
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 object FeatureFlags {
-    val customerSheetACHv2 = FeatureFlag()
+    val customerSheetACHv2 = FeatureFlag(enabledInDebugMode = true)
+    val forceAuthFlowV3 = FeatureFlag(enabledInDebugMode = false)
 }
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-class FeatureFlag {
+class FeatureFlag(
+    private val enabledInDebugMode: Boolean,
+) {
```

Comment:
maybe it makes sense to have `enabledInDebugMode` as a constructor parameter as well?